web/static/js/reducers/idea.js

```diff
@@ -1,5 +1,7 @@
 const idea = (state = [], action) => {
   switch (action.type) {
+    case "SET_IDEAS":
+      return [...action.ideas]
```

Comment:
@vanderhoop I like it! and 👍 to @kimberlysuazo's idea here. returning `action.ideas` outright seems the most straightforward approach, and will preserve immutability as it's a brand new array